pkgs/ffigen/test/native_objc_test/block_test.dart

```diff
@@ -260,6 +260,29 @@ void main() {
       expect(lib.getBlockRetainCount(rawBlock), 0);
     });
 
+    Pointer<Void> blockManualRetainRefCountTest() {
+      final block = IntBlock.fromFunction(lib, makeAdder(4000));
+      expect(lib.getBlockRetainCount(block.pointer.cast()), 1);
+      final rawBlock = block.retainAndReturnPointer().cast<Void>();
+      expect(lib.getBlockRetainCount(rawBlock.cast()), 2);
+      return rawBlock.cast();
+    }
+
+    blockManualRetainRefCountTest2(Pointer<Void> rawBlock) {
```

Comment:
Return type missing.

(The test would be easier to read if you return the count here and have the expect in the test itself.)

Reply:
Done